pom.xml

```diff
@@ -712,6 +712,12 @@
         <groupId>net.sourceforge.htmlunit</groupId>
         <artifactId>htmlunit</artifactId>
         <version>${htmlunit.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>xalan</groupId>
+            <artifactId>xalan</artifactId>
+          </exclusion>
+        </exclusions>
         <scope>test</scope>
```

Comment:
For the test dependency, we had better have `[TESTS]` tag.

Reply:
OK~ I'll remember to add it next time